proposals/artifacts.md

```diff
@@ -0,0 +1,56 @@
+Container registries, implementing the [distribution-spec][distribtuion-spec], provide reliable, highly scalable, secured storage services for container images. Customers either use a cloud provider implementation, vendor implementations, or instance the open source implementation of distribution. They configure security and networking to assure the images in the registry are locked down and accessible by the resources required. Cloud providers and vendors often provide additional values atop their registry implementations from security to productivity features. 
+
+Applications and services typically require additional artifacts to deploy and manage, including [helm](https://helm.sh) for deployment and [Open Policy Agent (OPA)](https://github.com/open-policy-agent/opa/issues/1413) for policy enforcement.
+
+Utilizing the [manifest][image-manifest] and [index][image-index] definitions, new artifacts can be stored and served using the [distribution-spec][distribution-spec] without changing the actual distribution spec. This repository will provide  a reference for artifact authors and registry implementors for supporting new artifact types with the existing implementations of distribution.
+
+By providing an OCI artifact definition, the community can continue to innovate, focusing on new artifact types without having to build yet another storage solution (yass). 
+
+## Proposal
+Under the http://github.com/opencontainers organization:
+
+- Create a new **artifacts** repository, named http://github.com/opencontainers/artifacts
+- Update [distribution][distribution-spec] to generically reference [manifest][image-manifest] and [index][image-index], with image as one of many artifact types it can store. 
+
+## Contents
+
+The repository will serve 2 primary goals:
+
+1. **artifact authors** - guidance for authoring new artifact types. Including a clearing house for well known artifact types.
+1. **registry operators and vendors** - guidance for how they can support new artifact types, including how they can opt-in or out of well known artifact types. 
+
+### Initial Maintainers
+Initial maintainers of the artifacts project would be :
+* Steve Lasker <steve.lasker@microsoft.com> (@stevelasker)
+* Derek McGowan <derek.mcgowan@docker.com> @derekmcgowan
+* Mike Brown <brownwm@us.ibm.com> @mikebrow
+
+### Code of Conduct
+This project would incorporate (by reference) the OCI [Code of Conduct][code-of-conduct].
+
+### Governance and Releases
+This project would incorporate the Governance and Releases processes from the OCI project template: https://github.com/opencontainers/project-template.
+
+### Project Communications
+Both of the proposed projects would continue to use existing channels in use by the OCI developer community for communication including:
+* GitHub for issues and pull requests
+* The dev@opencontainers.org email list
+* The weekly OCI developer community conference call
+* The #OpenContainers IRC channel
+
+### Versioning / Roadmap
```

Comment:
I would mention this repository will not be versioned, but will be continuously updated with a list of versioned types with historical references. As such, this repository will not have releases or an explicit roadmap, only a scope of what may be referenced.